### ISSUE TITLE
Delivery the collection for Spring & Liberty

### ISSUE THREAD

**Issue by dianak2016** [kabanero-io/kabanero-collection#46]

**Comment by marikaj123** [comment]
@ebullient - Please provide us with the link to the issue that would contain the fix.

**Comment by marikaj123** [comment]
Appsody stack needs to exist with respect to creating the application for the Sping Boot on Liberty.
1- Need to build the collection for Spring on Liberty.
2- Need to deliver the Tekton pipeline by the pipeline team. 
3- Need to deliver the appsody operator to deploy it.

**Comment by BarDweller** [comment]
Work on experimental Appsody Spring Boot2 on Liberty stack is under this PR https://github.com/appsody/stacks/pull/186

**Comment by ebullient** [comment]
Correction to @marikaj123's comment above:
There should be a generic tekton pipeline that uses the generic appsody operator.  There shouldn't be anything unique for Spring/Liberty that isn't defined/provided by the stack. 

In fairness, I believe there is one set of metadata that the appsody operator needs, but I think that's a bug/architectural issue that will have to be resolved in a different way.

In summary: 
Ozzy has an experimental appsody stack to satisfy (1)
There should be nothing unique to do for (2)
There may be metadata required for (3), but I disagree with how that metadata is defined now, and would like to see that fixed.

**Comment by marikaj123** [comment]
Blocked by Pipeline PR. Need two days after to complete. Collection team need some time to implement UBI image.

**Comment by ebullient** [comment]
This stack must remain experimental.  The experimental collection can be based on the Liberty UBI image

**Comment by ianpartridge** [comment]
It's not on the Kabanero delivery path, because we were told it was not part of Kabanero's initial release.

For it to get on the Kabanero delivery path it will need to move into `incubator`.

**Comment by groeges** [comment]
Removing from Epic as this is not for v0.1

**Comment by ebullient** [comment]
woops! you're right, Ian.. my brain ignored the '& Liberty' part.

**Comment by groeges** [comment]
@ebullient What additional work is needed on this stack to get it moved from experimental to incubator?  We need this to be in incubator to be able to use it in Kabanero!!

@ralanlittle I recall some mention on a playback of this stack and I think you mentioned that it wasn't part of Kabanero due to some licensing issues. Do you know what the status of the licensing is and whether it will be resolved in time for this stack to be part of the next release or do we need to pull it from v0.2 release?

**Comment by ebullient** [comment]
The issue is not licensing, it is support. We haven’t sorted support with RHR yet. Also, Alasdair and Mike have not seen enough value to push forward with Liberty/Spring integration (making it really smooth and well tested, e.g.). At the moment, I have no plans to work on this, and thought that we had tabled this requirement unless and until Mike and Alasdair say otherwise.
